docs/atlascli/command/atlas-clusters-failover.txt

```diff
@@ -0,0 +1,97 @@
+.. _atlas-clusters-failover:
+
+=======================
+atlas clusters failover
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Starts a failover test for the specified cluster in the specified project.
+
+Clusters contain a group of hosts that maintain the same data set. A failover test checks how MongoDB Cloud handles the failure of the cluster's primary node. During the test, MongoDB Cloud shuts down the primary node and elects a new primary.
+
+Syntax
+------
+
+.. code-block::
+
+   atlas clusters failover <clusterName> [options]
+
+.. Code end marker, please don't delete this comment
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterName
+     - string
+     - true
+     - Name of the cluster to start a failover test.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --force
+     - 
+     - false
+     - If specified, skips asking for confirmation before proceeding with a requested action.
```

Comment:
```suggestion
     - Flag that indicates whether to skip the confirmation prompt before proceeding with the requested action.
```

Reply:
I will address this, projectId and profile changes as a separate PR because it would trigger a regeneration on basically every single command